### DISCUSSION TITLE
i have a suggestion

### DISCUSSION THREAD

**Discussion by ghost** [DrunkOnBytes/flutter_collapsible_sidebar#33] [Ideas]
your sidebar is very nice working.
but when i tap widget.body i want sidebar to collapse.
so I tried modifying collapsible_sidebar code.

class CollapsibleSidebarState extends State<CollapsibleSidebar>
    with SingleTickerProviderStateMixin {
	//skip
  Widget build(BuildContext context) {
    **Widget page = GestureDetector(
      onTap: () {
        _isCollapsed = true;
        _animateTo(widget.minWidth);
      },
      child: widget.body,
    );** //added

    return Stack(
      alignment: Alignment.topLeft,
      children: [
        Padding(
          padding: EdgeInsets.only(left: widget.minWidth * 1.1),
          child: **page**, //edited
        ),
	//skip

it works as i expected
How about adding this feature?

**Comment by DrunkOnBytes** [comment]
Sure @partitionce, Just do a pull request with a comment explaining the new feature. I'll test it and then merge it to the repo.
